--- libgeotiff-1.1.4/Makefile.in.orig	Tue Dec 12 20:03:19 2000
+++ libgeotiff-1.1.4/Makefile.in	Mon Jun 17 23:53:29 2002
@@ -194,24 +194,24 @@
 	autoconf
 
 install:
-	$(INSTALL) -d $(libdir)
-	$(INSTALL) libgeotiff.a ${libdir}
-	$(INSTALL) -d $(includedir)
-	$(INSTALL) $(GT_INCLUDE_FILES) $(includedir)
-	$(INSTALL) -d $(datadir)
-	$(INSTALL) csv/*.csv $(datadir)
-	$(INSTALL) -d $(bindir)
-	$(INSTALL) bin/listgeo bin/geotifcp $(bindir)
+	$(INSTALL) -d $(DESTDIR)$(libdir)
+	$(INSTALL) libgeotiff.a $(DESTDIR)${libdir}
+	$(INSTALL) -d $(DESTDIR)$(includedir)
+	$(INSTALL) $(GT_INCLUDE_FILES) $(DESTDIR)$(includedir)
+	$(INSTALL) -d $(DESTDIR)$(datadir)
+	$(INSTALL) csv/*.csv $(DESTDIR)$(datadir)
+	$(INSTALL) -d $(DESTDIR)$(bindir)
+	$(INSTALL) bin/listgeo bin/geotifcp $(DESTDIR)$(bindir)
 	if test ! -z "$(GEOTIFF_SO)" ; then \
-	   $(INSTALL) $(GEOTIFF_SO) ${libdir}; \
-	   rm -f ${libdir}/libgeotiff.so.$(VERSION_MAJOR).$(VERSION_MINOR); \
-	   ${LN} ${libdir}/$(GEOTIFF_SO) ${libdir}/libgeotiff.so.$(VERSION_MAJOR).$(VERSION_MINOR); \
-	   rm -f ${libdir}/libgeotiff.so.$(VERSION_MAJOR); \
+	   $(INSTALL) $(GEOTIFF_SO) $(DESTDIR)${libdir}; \
+	   rm -f $(DESTDIR)${libdir}/libgeotiff.so.$(VERSION_MAJOR).$(VERSION_MINOR); \
+	   ${LN} ${libdir}/$(GEOTIFF_SO) $(DESTDIR)${libdir}/libgeotiff.so.$(VERSION_MAJOR).$(VERSION_MINOR); \
+	   rm -f $(DESTDIR)${libdir}/libgeotiff.so.$(VERSION_MAJOR); \
 	   ${LN} ${libdir}/libgeotiff.so.$(VERSION_MAJOR).$(VERSION_MINOR) \
-		${libdir}/libgeotiff.so.$(VERSION_MAJOR); \
-	   rm -f ${libdir}/libgeotiff.so; \
+		$(DESTDIR)${libdir}/libgeotiff.so.$(VERSION_MAJOR); \
+	   rm -f $(DESTDIR)${libdir}/libgeotiff.so; \
 	   ${LN} ${libdir}/libgeotiff.so.$(VERSION_MAJOR) \
-		${libdir}/libgeotiff.so; \
+		$(DESTDIR)${libdir}/libgeotiff.so; \
 	fi
 
 
